packages/components/psammead-most-read/src/List/index.jsx

```diff
@@ -34,25 +37,30 @@ const StyledGrid = styled(Grid).attrs({
   }
 `;
 
-const MostReadList = ({ numberOfItems, dir, children }) => (
-  <StyledGrid
-    {...mostReadListGridProps}
-    dir={dir}
-    numberOfItems={numberOfItems}
-    forwardedAs="ol"
-  >
-    {children}
-  </StyledGrid>
-);
+const MostReadList = ({ numberOfItems, dir, maxTwoColumns, children }) => {
+  const MostReadListGrid = maxTwoColumns ? StyledGridBase : StyledGridExtended;
```

Comment:
Just wondering if it would be easier to see what's going on if we passed maxTwoColumns as a prop to the StyledGrid, and then added a condition inside there? Instead of creating a separate 'Extended' version. Or maybe just renaming 'Base' and 'Extended' to 'TwoColumnGrid' and 'MultiColumnGrid' or something?

Reply:
> Or maybe just renaming 'Base' and 'Extended' to 'TwoColumnGrid' and 'MultiColumnGrid' or something?

I like this ^

Reply:
>  and then added a condition inside there? Instead of creating a separate 'Extended' version. 

originally I was going to do this, but when I added the condition logic within [/Rank/index.js](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-most-read/src/Rank/index.jsx#L75-L99), it looked a bit difficult to read.

Reply:
Cool, if renaming is the clearer option let's do that :)